lib/ionic_notification.rb

```diff
@@ -18,10 +18,26 @@ module IonicNotification
   mattr_accessor :ionic_app_in_production
   @@ionic_app_in_production = true
 
+  # Send notification even if it has no message
+  mattr_accessor :process_empty_messages
+  @@process_empty_messages = false
+
+  # Default message for notifications
+  mattr_accessor :default_message
+  @@default_message = "This was intended to be a beautiful notification. Unfortunately, you're not qualified to read it."
```

Comment:
Line is too long. [121/80]<br>Replace class var @@default_message with a class instance var.